### ISSUE TITLE
Update widget heading to H3 on Education letters

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#19958]
## Description
On the widget on the [Education letters page](https://www.va.gov/education/download-letters/), the widget should be updated to have an h3 heading instead of using the current `<p>` tag
![Image](https://github.com/user-attachments/assets/2321224c-601e-47f8-9711-217297ab0282)

### Scope / Impact analysis
_What, if anything, could break as a result of this change?_
_Engineer should assess this when approaching PR._

### Roles / assignments
_After functional testing, code review, accessibility review, and design review can happen in parallel._
- [ ] **Functional testing:** 
- [ ] **Code review:** 
- [ ] **Design review:** 
- [ ] **Accessibility review:** 
- [ ] **Product acceptance:** 

## Acceptance criteria
_ACs should capture expected behavior, to inform test plans. Consider devices, documentation updates including KBs, change management, and [content model](https://prod.cms.va.gov/admin/structure/cm_document) when applicable._

- [x] The widget heading "Sign in to download your VA education decision letter" is now using an h3 heading instead of using the current `<p>` tag
- [x] Accessibility review

**Comment by randimays** [comment]
Pulled in for Sprint 19.

**Comment by randimays** [comment]
Verified successfully in production; closing as complete.
